pkg/kubelet/prober/worker.go

```diff
@@ -152,9 +152,9 @@ func (w *worker) doProbe() (keepGoing bool) {
 	}
 
 	c, ok := api.GetContainerStatus(status.ContainerStatuses, w.container.Name)
-	if !ok {
+	if !ok || len(c.ContainerID) == 0 {
 		// Either the container has not been created yet, or it was deleted.
```

Comment:
I suppose it's possible that the pod was deleted, and the container status was removed before the probe worker was terminated. I think that's what the comment was about. We're splitting hairs here, but how about just going with a log message of "Probe target container not found" or something like that?